apps/charterafrica/payload.config.js

```diff
@@ -73,6 +73,7 @@ export default buildConfig({
       },
     }),
   },
+  cors: [process.env.PAYLOAD_ALLOWED_DOMAINS.split(",")],
```

Comment:
```suggestion
  cors,
```

... and do the processing just after [`const appURL = ...`](https://github.com/CodeForAfrica/ui/pull/325/files#diff-1b3a74b7cd022ec463c5322d5a323ff7d8624f32827817b84748c4401568844dR32)
```js
const cors = process?.env?.PAYLOAD_CORS?.split(",")
  ?.map((d) => d.trim())
  ?.filter(Boolean)
```

We want `cors` to be `undefined` if no URLs were set. We don't want it to be `[undefined]`.